trunner/runners/common.py

```diff
@@ -308,21 +310,46 @@ def go(self):
 class Runner:
     """Common interface for test runners"""
 
+    @abstractmethod
+    def status(self, status):
+        """Method for sygnalising a current runner status: busy/failed/succeeded"""
+        pass
+
+    @abstractmethod
     def flash(self):
         """Method used for flashing a device with the image containing tests."""
         pass
 
+    @abstractmethod
     def run(self, test):
         """Method used for running a single test case which is represented by TestCase class."""
         pass
 
 
 class DeviceRunner(Runner):
-    """This class provides interface to run test case using serial port"""
+    """This class provides interface to run tests on hardware targets using serial port
+       Hardware targets are run on Rpi
+       GPIO 13 must be connected to a red LED
+       GPIO 18 must be connected to a green LED
+       GPIO 12 must be connected to a blue LED"""
 
     def __init__(self, port):
         self.port = port
         self.serial = None
+        self.leds = {'red': GPIO(13), 'green': GPIO(18), 'blue': GPIO(12)}
```

Comment:
seems like class attribute, not instance one - but we might keep it as is if You think we will soon read leds configuration from external file dynamically when instantiating the class)

Reply:
I hope it will be soon, so I will keep it